man/systemd-bsod.xml

```diff
@@ -43,6 +43,7 @@
     <variablelist>
       <xi:include href="standard-options.xml" xpointer="help" />
       <xi:include href="standard-options.xml" xpointer="version" />
+      <xi:include href="standard-options.xml" xpointer="wait" />
```

Comment:
This is not a standard option, it's a custom one, so you will need to write a paragraph for it here describing it